entry_timeseries_common.go

```diff
@@ -49,41 +53,70 @@ func (t TsColumnInfo) Type() TsColumnType {
 	return t.kind
 }
 
+// Symtable : return column symbol table name
+func (t TsColumnInfo) Symtable() string {
+	return t.symtable
+}
+
 // NewTsColumnInfo : create a column info structure
 func NewTsColumnInfo(columnName string, columnType TsColumnType) TsColumnInfo {
-	return TsColumnInfo{columnName, columnType}
+	return TsColumnInfo{columnName, columnType, ""}
```

Comment:
We wouldn't need this helper function if we used named parameters. String would be zero-value-initialized (so with `""`). E.g.:
```
TsColumnInfo{name: columnName, kind: columnType}
```

See:
* https://golang.org/doc/effective_go.html#composite_literals
* https://stackoverflow.com/questions/12703243/what-is-the-zero-for-string

Reply:
I didn't know this feature, but I'm not sure I prefer this solution, because it would mean using literals everywhere for this constructor to be consistent ...

Reply:
Ok.